packages/openneuro-app/src/scripts/users/components/dataset-card.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react"
+import { formatDistanceToNow, parseISO } from "date-fns"
+import activityPulseIcon from "../../../assets/activity-icon.png"
+import { Tooltip } from "@openneuro/components/tooltip"
+import { Icon } from "@openneuro/components/icon"
+import styles from "../scss/datasetcard.module.scss"
+
+interface Dataset {
+  id: string
+  created: string
+  name: string
+  public: boolean
+  analytics: {
+    views: number
+    downloads: number
+  }
+  stars: [{ userId: string; datasetId: string }]
+  followers: [{ userId: string; datasetId: string }]
+  latestSnapshot?: {
+    id: string
+    size: number
+    issues: [{ severity: string }]
+    created?: string
+  }
+}
```

Comment:
Not that it needs to be changed in this PR but we should probably try to generate these types from the GraphQL schema as we're repeating this type definition quite a bit.

Reply:
I made user-types its own file and put it in the types dir.